src/app/core/utils/Input.ts

```diff
@@ -189,9 +201,6 @@ export class Input {
     public addListener(listener: Listener): void {
         this.listeners.push(listener);
     }
-    public removeListener(listener: Listener): void {
-        this.listeners.splice(this.listeners.indexOf(listener), 1);
-    }
```

Comment:
I'm not sure if this function was supposed to be deleted, when trying to run this branch locally I get an error because
```
Property 'removeListener' does not exist on type 'Input'.
```

Reply:
The main change of this file should only be the function I added and one change I added to touchManager. I am not sure why the rest of changes appear. I am gonna update the code, redo this change and try to make sure this function works without deleting others.